app/models/tag.rb

```diff
@@ -62,6 +63,11 @@ def self.contributor_count(tagname)
       uids+=n.revision.collect(&:uid)
     end
     uids = uids.uniq
+    uids
```

Comment:
Ah, so, do you think we should return an ActiveRecord collection here, instead of just uids? Like, replace this line with `User.find uids` instead -- and then the line above (https://github.com/publiclab/plots2/pull/2055/files#diff-a8eb2425ebeb0814d010864bb014e815R326) would be simpler -- make sense?

Reply:
Yes, makes sense, will do!!

Reply:
@jywarren done with the suggested changes!